chispa/dataframe_comparer.py

```diff
@@ -10,7 +10,7 @@ class DataFramesNotEqualError(Exception):
    pass
 
 
-def assert_df_equality(df1, df2, ignore_nullable = False, transforms=[]):
+def assert_df_equality(df1, df2, ignore_nullable = False, transforms = [], allow_nan_equality = False):
```

Comment:
Mutable default arguments in Python (like `[]`) are slightly dangerous. If you mutate `transforms` inside this function, the mutations will persist to the next call of the function.

The typical solution to this is to set the default to `None` and then inside the function convert to `[]`.

Reply:
Will address here: https://github.com/MrPowers/chispa/issues/17